open-api-spec.yaml

```diff
@@ -431,6 +415,142 @@ paths:
           in: query
           name: cursor
           description: The cursor key to list from
+  /quotes:
+    post:
+      summary: Create a Quote
+      tags: []
+      operationId: create-quote
+      responses:
+        '202':
+          description: Quote Created
+          headers:
+            Location:
+              schema:
+                type: string
+              description: The url of the created quote resource.
+        '401':
+          $ref: '#/components/responses/401'
+        '403':
+          $ref: '#/components/responses/403'
+      requestBody:
+        content:
+          application/json:
+            examples:
+              Create fixed-send-amount quote for $25:
+                value:
+                  receivingAccount: 'https://openpayments.guide/alice'
+                  sendAmount:
+                    amount: 2500
+              Create fixed-receive-amount quote for $25:
+                value:
+                  receivingPayment: 'https://openpayments.guide/alice/incoming-payments/37a0d0ee-26dc-4c66-89e0-01fbf93156f7'
+                  receiveAmount:
+                    amount: 2500
+            schema:
+              type: object
+              properties:
+                receivingAccount:
+                  type: string
+                  format: uri
+                  description: The URL of the account that is being paid.
```

Comment:
Should a quote auto-create the corresponding incoming payment?
I think ilp-pay would need to be updated to be able to quote for a `receivingAccount` without creating an incoming payment.

Reply:
Auto-create for outgoing payment, not quote.